### DISCUSSION TITLE
Message d'erreur au lancement du script (version 26 03 2023)

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#1863] [Q&A]
bonjour, sur la toute dernière version téléchargée ce jour , ça ne va pas jusqu'au bout et 
j'ai un message d'erreur a l'étape "Start Stable-Diffusion"

Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 15, in <module>
    from modules import paths, timer, import_hook, errors
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/paths.py", line 4, in <module>
    from modules.paths_internal import models_path, script_path, data_path, extensions_dir, extensions_builtin_dir
ModuleNotFoundError: No module named 'modules.paths_internal'

**Comment by TheLastBen** [comment]
ton sd n'est pas à jour, git pull ne marche pas apparemment dans ton cas

**Reply by darth2602** [reply]
j'ai téléchargé la toute dernière version du script ....  peu être en supprimant le dossier SD de google drive ?

**Reply by TheLastBen** [reply]
ton git pull ne marche pas donc il n'a pas téléchargé le script paths_internal.py qui se retrouve dans modules

**Reply by darth2602** [reply]
j'ai supprimé le dossier sd et relancé le script, ça a l'air de marcher (pas tout vérifié) 

j'ai quand même un message d'erreur au démarrage : 


Error loading script: lora_script.py
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/scripts.py", line 256, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/script_loading.py", line 11, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions-builtin/Lora/scripts/lora_script.py", line 4, in <module>
    import lora
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/extensions-builtin/Lora/lora.py", line 238, in <module>
    def lora_apply_weights(self: torch.nn.Conv2d | torch.nn.Linear | torch.nn.MultiheadAttention):
TypeError: unsupported operand type(s) for |: 'type' and 'type'

Loading weights [4c86efd062] from /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Stable-diffusion/model.ckpt
Creating model from config: /content/gdrive/MyDrive/sd/stable-diffusion-webui/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Applying xformers cross attention optimization.
Textual inversion embeddings loaded(0): 
Model loaded in 9.9s (load weights from disk: 5.0s, create model: 0.7s, apply weights to model: 2.0s, apply half(): 1.2s, move model to device: 1.0s).
Running on local URL: https://xxxxxxxxxxxxxxxxxxxxxxxxxx.lt/
✔ Connected
Startup time: 20.7s (import torch: 4.5s, import gradio: 0.8s, import ldm: 0.2s, other imports: 1.6s, list extensions: 0.8s, load scripts: 1.9s, load SD checkpoint: 9.9s, create ui: 0.6s).

**Reply by darth2602** [reply]
Donc oui , malgré le message a l'air de fonctionner, par contre il y a un problème avec le preview, je vois la progression du calcul (j'ai réglé avec une progression de 5 images) par contre l'image finale n'apparait pas - mais elle est quand même sauvegardée.... étrange....

**Reply by darth2602** [reply]
j'ai décoché use_local_tunnel et des fois ca affiche l'image finale, des fois non

**Reply by AiXplorer777** [reply]
Cela l'a corrigé pour moi aussi. Merci!

**Reply by TheLastBen** [reply]
Utilise ngrok, il est mieux que localtunnel et gradio server

**Reply by meunumerotim333** [reply]
ngrok is also giving lora error and other errors too like picture never send to IMG2IMG tab, always need to do manually

**Reply by TheLastBen** [reply]
ngrok fixes the issue with freezing, lora error was fixed, and the other issues are exclusive to the A1111 repo
